### ISSUE TITLE
make a `brew` formula

### ISSUE THREAD

**Issue by zramsay** [eris-ltd/eris-cli#560]
because why not. low priority

**Comment by compleatang** [comment]
we have a repo for this but its super old. github.com/eris-ltd/homebrew-eris

**Comment by zramsay** [comment]
sweet. nice starting point for sure

**Comment by pietv** [comment]
great idea!

**Comment by pietv** [comment]
https://github.com/Homebrew/homebrew-core/pull/1180

(Man page generation pending on issue #712.)
